src/workspacer.Bar/Widgets/ActiveLayoutWidget.cs

```diff
@@ -9,6 +9,7 @@ namespace workspacer.Bar.Widgets
 {
     public class ActiveLayoutWidget : BarWidgetBase
     {
+        public string FontName { get; set; } = null;
```

Comment:
Any reason not to just put this in BarWidgetBase, if all widgets are going to have this?

Reply:
Blanked on the fact that this was possible... Thanks for pointing it out!